iothub/device/src/ConnectionStatus.cs

```diff
@@ -32,7 +32,7 @@ public enum ConnectionStatus
         /// The device connection was closed gracefully.
         /// </summary>
         /// <remarks>
-        /// To perform more operations on the client, call <see cref="IotHubDeviceClient.Dispose()"/> and then re-initialize the client.
```

Comment:
Cannot resolve the inherited method in doc comments. This is supposed to link to IotHubDeviceClient.Dispose() and IotHubModuleClient.Dispose().